extensions/amp-experiment/0.1/variant.js

```diff
@@ -39,7 +39,7 @@ export function allocateVariant(win, config) {
 
   if (cidScope && config.consentNotificationId) {
     hasConsentPromise = userNotificationManagerFor(win)
-        .then(manager => manager.get(config.consentNotificationId))
+        .then(manager => manager.getNotification(config.consentNotificationId))
```

Comment:
@jridgewell I think this promise waits for both doc ready and viewer is visible, that wouldn't be enough?